.eslintrc.js

```diff
@@ -1,21 +0,0 @@
-'use strict'
-
-module.exports = {
-  extends: 'standard',
-  env: {
-    mocha: true,
-    node: true
-  },
-  rules: {
-    'padded-blocks': ['error', {
-      blocks: 'never',
-      switches: 'never',
-      classes: 'always'
-    }],
-    'space-before-function-paren': ['error', {
-      anonymous: 'always',
-      named: 'never',
-      asyncArrow: 'always'
-    }]
-  }
-}
```

Comment:
commit message is wrong, should say: 'remove eslint' should be squashed into the commit that removes it from package.json too